tests/CommandTest.php

```diff
@@ -93,6 +93,7 @@ public function testRequiredOptions()
     /**
      * @expectedException \Equip\Command\CommandException
      * @expectedExceptionCode \Equip\Command\CommandException::MISSING_OPTION
+     * @expectedExceptionRegExp /required options not defined:/i
```

Comment:
Unless I'm missing something, it would be nice to see a test that lists of the missing options. 
```
required options not defined: `user_id`, `article_id`
```